plugins/backup/backup-manager

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
```

Comment:
Please add a POD-style documentation header here. See the plugin [processes/multimemory](https://github.com/munin-monitoring/contrib/blob/master/plugins/processes/multimemory) for a good example.
Also add the author, a licence and the capabilities.